db/migrate/20150107044854_users_without_location_cleanup.rb

```diff
@@ -0,0 +1,9 @@
+class UsersWithoutLocationCleanup < ActiveRecord::Migration
+  def up
+    User.delete_all(location: [nil,''])
```

Comment:
@rohitpaulk what in your opinion do you think root this issue? @seuros is there possibility that it should be the kind like endorsements that we closed y'day?